### PR TITLE
Add fonts

### PR DESCRIPTION
**Pull request by auaruss**
## Description

Adds the fonts to the project.

See Story: [FSA21V2-136](https://sparkbox.atlassian.net/browse/FSA21V2-136)

## To Test

1. Make sure all PR Checks have passed (Github Actions, Netlify etc).
2. Pull down all related branches.
3. Confirm all tests pass: `npm run test:ci` and `npm run lint`.

## Validation

The following has been completed by the developer:

_[delete anything irrelevant to this PR]_

- [x] This PR has code changes, and our linters still pass.
- [x] This PR affects production code, so it was browser tested (see below).

### Browser Testing

_[delete if irrelevant to this issue]_

#### Gold Level Browsers

In these browsers, all content is accessible and design matches exactly. In most cases, bugs should be resolved before merge.

**macOS**

- [x] Chrome, current release
- [x] Firefox, current release
- [x] Safari, current release

**Windows**

- [x] Chrome, current release
- [x] Firefox, current release

**Mobile**

- [x] Chrome on Android 9
- [x] Safari, current release, on iPhone

#### Silver Level Browsers

In these browsers, all content is readable, but design may be inaccurate.

**Windows**

- [x] Edge, current release

**Mobile**

- [x] Android 8 - Chrome on Phone
- [x] Safari, version before current release, on iPad (including various split screen widths)
- [x] Chrome on iPhone

#### Bronze Level Browsers

In these browsers, our only goal is content readability.

**Windows**

- [x] IE 11

### CODE REVIEW

**Review by catheraaine — COMMENTED** [review]
## We've got fonts!!! 

I've got a few recommendations on the PR in comments. Additionally, can we rename the zip files to be all lower case with no spaces in the file names? 

👍  Excellent work so far.

File: styles/_fonts.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// fonts needed
```
[the remark this answers is not in the document]
Comment (auaruss) [reply styles/_fonts.scss:1]: Fixed.

File: styles/_fonts.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+// Sole Sans VF
+// Sole Sans
+// Sole Serif Big Display
+
```
[the remark this answers is not in the document]
Comment (auaruss) [reply styles/_fonts.scss:5]: Fixed.

File: pages/components.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ const Components = () => {
   return (
     <main>
       <Hero />
+      <ApprenticeQualities />
       <h1>Components</h1>
```
[the remark this answers is not in the document]
Comment (auaruss) [reply pages/components.js:19]: Addressed, will follow up on slack.

File: styles/_fonts.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+ * Webfonts copyright: &amp;#x00A9; 2018 CAST Cooperativa Anonima Servizi Tipografici. All rights reserved.
+ *
+ * © 2020 MyFonts Inc
+*/
+
+@font-face {
+  font-family: "Sole Sans";
+  src:
+    url("./fonts/sole-sans-vf.woff2") format("woff2"),
+    url("./fonts/sole-sans-vf.woff") format("woff");;
```
Comment (catheraaine) [inline styles/_fonts.scss:30]: this one still needs a weight and style

File: styles/global.scss [hunk 5]
```diff
@@ -1,3 +1,8 @@
+@use "fonts";
+
+$font-default:  sole sans, 'Trebuchet MS', -apple-system, blinkmacsystemfont, helvetica neue, sans-serif;
```
Comment (catheraaine) [inline styles/global.scss:3]: excellent

**Review by catheraaine — APPROVED** [review]
Excellent!

a grumpy space cat and I do not know why

**Pull request merged**